bpf/bpf_host.c

```diff
@@ -918,6 +926,12 @@ int from_host(struct __ctx_buff *ctx)
 	return handle_netdev(ctx, true);
 }
 
+/*
+ * to-netdev is attached as a tc egress filter to one or more physical devices
+ * managed by Cilium. (eg. eth0) This program is only attached when:
```

Comment:
Nit: 
```suggestion
 * managed by Cilium (e.g., eth0). This program is only attached when:
```

Reply:
Thanks, addressing this in a separate patch. :+1: